### PR TITLE
Add homepage to plugin details

### PR DESCRIPTION
**Pull request by gergo85**

### CODE REVIEW

**Comment by SaifurRahmanMohsin** [comment]
Thank you for the PR.

File: Plugin.php [hunk 1]
```diff
@@ -1,14 +1,13 @@
 <?php namespace Mohsin\Txt;
 
-use Backend;
 use System\Classes\PluginBase;
+use Backend;
 
 /**
```
Comment (Danielberry90) [inline Plugin.php]: Open

Comment (SaifurRahmanMohsin) [inline Plugin.php]: What do you mean?

Comment (Danielberry90) [inline Plugin.php]: D

**Pull request merged**